preliz/distributions/continuous.py

```diff
@@ -126,7 +126,7 @@ def _update(self, lam):
         self.params = (self.lam,)
         self._update_rv_frozen()
 
-    def fit_moments(self, mean, sigma=None):  # pylint: disable=unused-argument
+    def _fit_moments(self, mean, sigma=None):  # pylint: disable=unused-argument
```

Comment:
If sigma is not used why not do this to make things very obvious?
```suggestion
    def _fit_moments(self, mean, _=None):  # pylint: disable=unused-argument
```

Reply:
In other parts of the code like in maxent function we call this generally as distribution._fit_moments(self, mean, sigma). So this avoids the need to handle special cases in those other parts.